coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -43,6 +43,10 @@
         'file_arg_template': '+{line},{column} {filename} ',
         'gui': False
     },
+    'emacs': {
```

Comment:
These entries have no corresponding tests, right?
Did you try this locally and it worked?